crates/volta-core/src/project/mod.rs

```diff
@@ -171,13 +171,15 @@ impl Project {
         })
     }
 
-    /// Does this project use Yarn Plug'n'Play?
-    // (project uses Yarn, and either of the files '.pnp.js' or '.pnp.cjs' exist)
-    pub fn is_yarn_pnp(&self) -> bool {
+    /// Does this project use Yarn Berry?
+    // (project uses Yarn berry as detected by the files 'yarnrc.yml', '.pnp.js' or '.pnp.cjs' exist)
+    pub fn is_yarn_berry(&self) -> bool {
```

Comment:
very minor: Since this also applies to using PnP in Yarn 1.0, not just Yarn Berry, would it make sense to make this name more generic? Perhaps something like `needs_yarn_run`?

Reply:
Updated